CorsixTH/Lua/hospital.lua

```diff
@@ -1106,9 +1111,9 @@ function Hospital:createEmergency(emergency)
       },
     }
     self.world.ui.bottom_panel:queueMessage("emergency", message, nil, Date.hoursPerDay() * 16, 2) -- automatically refuse after 16 days
-    created_one = true
+    return -- sucessfully created
   end
-  return created_one
+  return "no heliport"
```

Comment:
This error is currently a catch all that includes staff desk, heliport, and according to the prompt 'not yet discovered' though you have a new condition for that.
You may break out a new error for the staff desk too?
Either way it's best to exit the function up when you know the condition doesn't hold rather than indenting the whole body. (Not a new issue)

Reply:
The new original prompt actually only happens if we have no diseases discovered at all. I'll need to revisit the code again to see if this actually happens outside level 1 (and maybe 2)